core/modules/views_ui/css/views_ui.admin.theme.css

```diff
@@ -232,6 +271,46 @@ input.form-radio {
   font-weight: bold;
 }
 
+@media (max-width: 768px) {
+  table.sticky-header {
+    display: none;
+  }
+
+  [id="views-list"] thead {
```

Comment:
Sounds good. Yeah, the attribute selector is as strong as a class, while ID's are a pwnhammer.